src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-packaging-form/sw-product-packaging-form.html.twig

```diff
@@ -1,124 +1,34 @@
 {% block sw_product_packaging_form %}
 <div class="sw-product-packaging-form">
+    <p class="sw-product-packaging-form__description">
+        {{ $tc('sw-product.packagingForm.description') }}
+    </p>
+
+    <!-- @deprecated tag:v6.8.0 - <sw-container> will be removed -->
     <sw-container
+        v-if="false"
```

Comment:
I would keep this solution to keep the indents visible. After this is done, I will definitely create a chore PR to delete these lines of code.